larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -1081,6 +1083,8 @@ StatusCode MasterAlgorithm::RegisterCustomContent(const Pandora *const /*pPandor
 
 StatusCode MasterAlgorithm::ReadSettings(const TiXmlHandle xmlHandle)
 {
+    PANDORA_RETURN_RESULT_IF(STATUS_CODE_SUCCESS, !=, XmlHelper::ReadValue(xmlHandle, "LArCaloHitVersion", m_larCaloHitVersion));
```

Comment:
Suggest move this down to around line ~1193, so that ReadSettings implementation still starts by focusing on  the fiddly external settings work.